style.css

```
@@ -28,3 +28,11 @@ a {
 	outline-width: 2px;
 	outline-style: solid;
 }
+
+/*
+ * Progresive enhancement to reduce widows and orphans
+ * https://github.com/WordPress/gutenberg/issues/55190
+ */
+h1, h2, h3, h4, h5, h6, blockquote, caption, figcaption, p {
+	text-wrap: pretty;
+}
```

```suggestion
}

```
There needs to be a single blank line at the end of the document.